templates/ocs-deployment/cephblockpool.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: {{ rook_api_version }}
+kind: CephBlockPool
+metadata:
+  name: {{ cephblockpool_name | default('ocsci-cephblockpool') }}
+  namespace: {{ cluster_namespace | default('openshift-storage') }}
+spec:
+  failureDomain: {{ failureDomain | default('host') }}
+  replicated:
+    size: {{ size | default('2') }}
```

Comment:
I am thinking this is OSD replica, If so, the default recommended is '3'

Reply:
Please change the param name from "size" to "replicasize"

Reply:
changed to replica_count